dandi/files/zarr.py

```diff
@@ -180,6 +182,9 @@ def get_metadata(
     ) -> BareAsset:
         metadata = get_default_metadata(self.filepath, digest=digest)
         metadata.encodingFormat = ZARR_MIME_TYPE
+        # TODO: .size obtained via get_default_metadata would be the one
+        # from os.stat and thus not reflective of actual size of the zarr
+        # folder.
```

Comment:
Actually, for Zarrs, the size is extracted from the digest:

https://github.com/dandi/dandi-cli/blob/4920831257762c4bb039db2f682ae24549292c68/dandi/metadata.py#L993-L998

Reply:
oh, missed that, thanks! I will remove the TODO but might RF to move this zarr specific code into zarr specific place ;)